src/js/pages/Index.js

```diff
@@ -150,15 +151,32 @@ var Index = React.createClass({
     });
 
     return (
-      <div className={classSet}>
-        {this.getScreenOverlays(showErrorScreen)}
-        <Sidebar location={this.props.location} />
-        {this.props.children}
-        <Modals
-          showErrorModal={this.state.showErrorModal}
-          modalErrorMsg={this.state.modalErrorMsg}
-        />
-        <ServerErrorModal />
+      <div
+        className={classSet}
+        style={{
```

Comment:
Why do we need to use inline styles here?

Reply:
because I had no idea how to deal with CNVS. styling there is such a mess and as we want to move to emotion with colocated styles anyway, I felt that having it as inline styles isnt _that_ bad. 🤷‍♂️ 

is that something you want me to change? or just a question?

Reply:
if it's a candidate for the ui-kit, shall we extract the styles the ui-kit way?

Reply:
Isn't this the index page? Why is it a ui-kit candidate?

Reply:
I would really like the styles being moved to CSS

Reply:
Please extract inline styles to `.application-wrapper`.